Source/WebInspectorUI/UserInterface/Models/CSSKeywordCompletions.js

```diff
@@ -230,6 +229,9 @@ WI.CSSKeywordCompletions.forFunction = function(functionName)
         suggestions.pushAll(WI.CSSKeywordCompletions._colors);
     }
 
+    if (typeof additionalFunctionValueCompletionsProvider === "function")
```

Comment:
NIT: we probably never expect `additionalFunctionValueCompletionsProvider` to be anything other than a function, so I think we should just have it be `if (additionalFunctionValueCompletionsProvider)`